packages/demos-babylon/webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path');
+const CopyPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  devtool: 'eval-source-map',
+  resolve: {
+    modules: ['node_modules'],
+  },
+  plugins: [
+    new CopyPlugin({
+      patterns: [
+        {
+          from: '**/*.*',
+          context: 'src/',
+          globOptions: { ignore: ['**/*.js' ]}
+        }
+      ]
+    }),
+  ],
+  entry: {
+    helloWorldDemo: './src/helloWorldDemo.js',
+    gesturesDemo: './src/gesturesDemo.js',
+    customCharacterDemo: './src/customCharacterDemo.js',
+  },
+  output: {
+    filename: '[name].bundle.js',
+    path: path.resolve(__dirname, 'dist')
+  },
+  devServer: {
+    open: ['/'],
+    liveReload: true,
+    hot: false,
```

Comment:
any particular reason this is false?

Reply:
In the next update you review you'll see I've gone with default settings where possible.